daemon/graphdriver/devmapper/mount.go

```diff
@@ -59,6 +59,7 @@ func ProbeFsType(device string) (string, error) {
 	buffer := make([]byte, maxLen)
 	l, err := file.Read(buffer)
 	if err != nil {
+		file.Close()
```

Comment:
@MabinGo usage of `defer` is idiomatic in go. it provides certain guarantees: http://blog.golang.org/defer-panic-and-recover it's no big deal to add defer in these cases. I'm not sure what we gain by preserving original style. It was missing file.Close() and you can add them using `defer`.